src/Umbraco.Cms.Api.Management/Controllers/Webhook/CreateWebhookController.cs

```diff
@@ -0,0 +1,50 @@
+using Asp.Versioning;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Umbraco.Cms.Api.Management.ViewModels.Webhook;
+using Umbraco.Cms.Core;
+using Umbraco.Cms.Core.Mapping;
+using Umbraco.Cms.Core.Models;
+using Umbraco.Cms.Core.Security;
+using Umbraco.Cms.Core.Services;
+using Umbraco.Cms.Core.Services.OperationStatus;
+using Umbraco.Cms.Core.Webhooks;
+using Umbraco.Cms.Web.Common.Authorization;
+
+namespace Umbraco.Cms.Api.Management.Controllers.Webhook;
+
+[ApiVersion("1.0")]
+[Authorize(Policy = "New" + AuthorizationPolicies.TreeAccessWebhooks)]
+public class CreateWebhookController : WebhookControllerBase
+{
+    private readonly IWebhookService _webhookService;
+    private readonly IUmbracoMapper _umbracoMapper;
+    private readonly IBackOfficeSecurityAccessor _backOfficeSecurityAccessor;
+
+    public CreateWebhookController(
+        IWebhookService webhookService,
+        IUmbracoMapper umbracoMapper,
+        IBackOfficeSecurityAccessor backOfficeSecurityAccessor)
+    {
+        _webhookService = webhookService;
+        _umbracoMapper = umbracoMapper;
+        _backOfficeSecurityAccessor = backOfficeSecurityAccessor;
+    }
+
+    [HttpPost]
+    [MapToApiVersion("1.0")]
+    [ProducesResponseType(StatusCodes.Status201Created)]
+    [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status404NotFound)]
+    public async Task<IActionResult> Create(CreateWebhookRequestModel createWebhookRequestModel)
+    {
+        Core.Models.Webhook created = _umbracoMapper.Map<Core.Models.Webhook>(createWebhookRequestModel)!;
+
+        Attempt<IWebhook, WebhookOperationStatus> result = await _webhookService.CreateAsync(created); //, CurrentUserKey(_backOfficeSecurityAccessor));
```

Comment:
I'd rather have TODOs in the service telling us to implement audit logging for this 😁 
```suggestion
        Attempt<IWebhook, WebhookOperationStatus> result = await _webhookService.CreateAsync(created);
```